Makefile

```diff
@@ -0,0 +1,49 @@
+TAG     := $(shell git rev-parse --short HEAD)
+SERVER  := fosskers/gtsite-service
+NGINX   := fosskers/gtsite-nginx
+TRANSIT := fosskers/gt-transit
+CHATTA  := fosskers/gt-chatta-demo
+
+clean:
+	cd geotrellis-site && make clean
+	cd geotrellis-chatta-demo && make clean
+	cd geotrellis-transit && ./sbt clean
+
+# One-time data fetches/ingests
+setup: chatta
```

Comment:
So long as keeping track of the individual commit hashes isn't more pain than it's worth. I'm still pondering this.